src/style.css

```
@@ -0,0 +1,91 @@
+@font-face {
+  font-family: Avenir ;
+  src: url(fonts/Avenir-Book.ttf);
+  font-style: normal;
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: Avenir ;
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-style: normal;
+  font-weight: 600;
+  color: var(--html-color);
+}
+
+html {
+  font-family: Avenir, Helvetica, sans-serif;
+  font-weight: 300;
+
+  --html-color: #3d4e61;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+.searchBar {
+  margin-top: 20px ;
+  border: 1px solid #e1e7ed;
+  border-radius: 4px;
+  height: 70px;
+  position: relative;
+}
+
+.searchBarS {
```

```suggestion
.searchBar-small {
```
it's better to use kebab-case for class naming